infra/scripts/test_android_runtime_release.sh

```diff
@@ -0,0 +1,140 @@
+ANDROID_REPORT_DIR="$ANDROID_WORKDIR/report"
+
+# Model download server setting
+if [[ -z "${MODELFILE_SERVER}" ]]; then
+  echo "[ERROR] Model file server is not set"
+  echo "        Need to download model file for test"
+  exit 1
+else
+  echo "Model Server: ${MODELFILE_SERVER}"
+fi
+
+apt-get update && apt-get install -y curl
+
+# Download Model
+$ROOT_PATH/tests/scripts/models/run_test.sh --download=on --run=off
+$ROOT_PATH/Product/aarch64-android.release/out/test/models/run_test.sh --download=on --run=off \
+  --configdir=$ROOT_PATH/Product/aarch64-android.release/out/test/models/nnfw_api_gtest \
+  --cachedir=$ROOT_PATH/Product/aarch64-android.release/out/unittest_standalone/nnfw_api_gtest_models
+
+N=`adb devices 2>/dev/null | wc -l`
+
+# exit if no device found
+if [[ $N -le 2 ]]; then
+    echo "No device found."
+    exit 1;
+fi
+
+NUM_DEV=$(($N-2))
+echo "device list"
+DEVICE_LIST=`adb devices 2>/dev/null`
+echo "$DEVICE_LIST" | tail -n"$NUM_DEV"
+
+if [ -z "$SERIAL" ]; then
+    SERIAL=`echo "$DEVICE_LIST" | tail -n1 | awk '{print $1}'`
+fi
+ADB_CMD="adb -s $SERIAL "
+
+# root on, remount as rw
+$ADB_CMD root on
+$ADB_CMD shell mount -o rw,remount /
+
+$ADB_CMD shell rm -rf $ANDROID_WORKDIR
+$ADB_CMD shell rm -rf /data/local/tmp/TestCompilationCaching*
+$ADB_CMD shell mkdir -p $ANDROID_REPORT_DIR
+$ADB_CMD push $ROOT_PATH/tests $ANDROID_WORKDIR/.
+$ADB_CMD push $ROOT_PATH/Product/aarch64-android.release/out $ANDROID_WORKDIR/Product/.
+
+# TFloader Testing
+TESTLIST=$(cat "${ROOT_PATH}/Product/aarch64-android.release/out/test/list/tflite_loader_list.${TEST_ARCH}.txt")
+$ADB_CMD shell LD_LIBRARY_PATH=$ANDROID_WORKDIR/Product/lib BACKENDS=acl_cl sh $ANDROID_WORKDIR/tests/scripts/models/run_test_android.sh \
+                                                        --driverbin=$ANDROID_WORKDIR/Product/bin/tflite_loader_test_tool \
+                                                        --reportdir=$ANDROID_REPORT_DIR \
+                                                        --tapname=tflite_loader.tap ${TESTLIST:-}
+# Union SkipList and testing List Creation
+UNION_MODELLIST_PREFIX="${ROOT_PATH}/Product/aarch64-android.release/out/test/list/frameworktest_list.${TEST_ARCH}"
+UNION_SKIPLIST_PREFIX="${ROOT_PATH}/Product/aarch64-android.release/out/unittest/nnapi_gtest.skip.${TEST_PLATFORM}"
+sort $UNION_MODELLIST_PREFIX.${BACKENDS[0]}.txt > $UNION_MODELLIST_PREFIX.intersect.txt
+sort $UNION_SKIPLIST_PREFIX.${BACKENDS[0]} > $UNION_SKIPLIST_PREFIX.union
+for BACKEND in "${BACKENDS[@]:1}"; do
+  comm -12 <(sort $UNION_MODELLIST_PREFIX.intersect.txt) <(sort $UNION_MODELLIST_PREFIX.$BACKEND.txt) > $UNION_MODELLIST_PREFIX.intersect.next.txt
+  comm <(sort $UNION_SKIPLIST_PREFIX.union) <(sort $UNION_SKIPLIST_PREFIX.$BACKEND) | tr -d "[:blank:]" > $UNION_SKIPLIST_PREFIX.union.next
+  mv $UNION_MODELLIST_PREFIX.intersect.next.txt $UNION_MODELLIST_PREFIX.intersect.txt
+  mv $UNION_SKIPLIST_PREFIX.union.next $UNION_SKIPLIST_PREFIX.union
+done
```

Comment:
Not tested. Maybe more readable.
(suggestion)
```bash
# Intersection of framework test list files
UNION_MODELLIST_PREFIX=( "${BACKENDS[@]/#/${ROOT_PATH}/Product/aarch64-android.release/out/test/list/frameworktest_list.${TEST_ARCH}.}" )
UNION_MODELLIST_PREFIX=( "${UNION_MODELLIST_PREFIX[@]/%/.txt}" )

join "${UNION_MODELLIST_PREFIX[@]}" > "${UNION_MODELLIST_PREFIX}.intersect.txt"

# Union of skip list files
UNION_SKIPLIST_PREFIX=( "${BACKENDS[@]/#/${ROOT_PATH}/Product/aarch64-android.release/out/unittest/nnapi_gtest.skip.${TEST_PLATFORM}.}" )

sort -u "${UNION_SKIPLIST_PREFIX[@]}" > "${UNION_SKIPLIST_PREFIX}.union"
```

Reply:
@mhs4670go I changed a little that you suggested. Thanks for pointing out!